app/templates/admin/createEvent.html

```diff
@@ -227,11 +227,16 @@ <h5 class="modal-title" id="warningModel">Delete Event</h5>
       </div>
       <div class="modal-footer">
         <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+        {% if eventData.recurringId != None %}
```

Comment:
This would be better in a positive statement. Instead try {% if eventData.recurringId %} should give  you the same result without the '!='